src/dotnet/UI.Blazor.App/Components/AudioPlayer/AudioTrackPlayer.cs

```diff
@@ -129,7 +129,7 @@ protected override async ValueTask ProcessMediaFrame(MediaFrame frame, Cancellat
                     throw StandardError.StateTransition(GetType(), "Can't process media frame before initialization.");
 
                 var chunk = frame.Data;
-                _ = _jsRef.InvokeVoidAsync("frame", cancellationToken, chunk);
+                await _jsRef.InvokeVoidAsync("frame", cancellationToken, chunk).ConfigureAwait(false);
```

Comment:
We should not await there, in this case we will have to wait for full round trip for each 20ms audio frame - it will ruin audio streaming in Blazor Server mode

Reply:
fixed